src/libs/antares/study/parts/short-term-storage/container.cpp

```diff
@@ -21,7 +21,6 @@
 
 #include <antares/logs/logs.h>
 #include <yuni/io/file.h>
```

Comment:
**Question** : 

if we replace : 
```C++
Yuni::IO::Directory::Create(folder);
```
with : 
```C++
std::filesystem::create_directories(folder);
```
, could we then remove the `#include <yuni/io/file.h>` ?